src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPrefooters.java

```diff
@@ -14,9 +15,14 @@
       dataProvider = "adsMiddlePrefooter",
       groups = "AdsMiddlePrefooter"
   )
-  public void adsMiddlePrefooter(String wikiName, String path, boolean middlePrefooterEnabled) {
+  public void adsMiddlePrefooter(
+      String wikiName,
+      String path,
+      Dimension windowResolution,
+      boolean middlePrefooterEnabled
+  ) {
     String testedPage = urlBuilder.getUrlForPath(wikiName, path);
-    AdsBaseObject adsBaseObject = new AdsBaseObject(driver, testedPage);
+    AdsBaseObject adsBaseObject = new AdsBaseObject(driver, testedPage, windowResolution);
```

Comment:
some VM just ain't capable of displaying browser in resolution wider than their monitors area.
It is known issue that some tests failed because of that. I think this issue should also be solved on the level of virtual machines so that a machine will not execute test that requires resolution wider/higher than the machine capacity.
I also believe we should find some common repository-wide approach for handling window resolution